cmd/manager/main.go

```diff
@@ -10,7 +10,7 @@ import (
 	"strings"
 	"time"
 
-	api "github.com/codeready-toolchain/api/pkg/apis"
+	api "github.com/codeready-toolchain/api/api/v1alpha1"
```

Comment:
same comment as in https://github.com/codeready-toolchain/toolchain-common/pull/177#discussion_r636104231 regarding the alias for the `github.com/codeready-toolchain/api/api/v1alpha1` package

Reply:
That would require a massive refactoring and since this PR doesn't change any aliases let's maybe keep it as is in this PR and do this refactoring later?

Reply:
I already did it in other PRs and since it's just a matter of applying a few `sed` commands, then it's fine to do it now: 6b734e8